.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+# Auto formats python code
+repos:
+
+-    repo: https://github.com/psf/black.git
+     rev: 23.1a1
```

Comment:
The latest official release is 22.12 per the repo, so I'm cool changing to that one instead so we don't have a pre-release version tag hanging out here.